keyboards/handwired/co60/rev1/rev1.h

```diff
@@ -0,0 +1,127 @@
+) \
+{ \
+    { K000, K001,   K002, K003,  K004, K005,  K006, K007,  K008, K009,  K010, K011, K012,  K013, K014 }, \
+    { K100, KC_NO,  K102, K103,  K104, K105,  K106, K107,  K108, K109,  K110, K111, K112,  K113, K114 }, \
+    { K200, KC_NO,  K202, K203,  K204, K205,  K206, K207,  K208, K209,  K210, K211, K212,  K213, K214 }, \
+    { K300, K301,   K302, K303,  K304, K305,  K306, K307,  K308, K309,  K310, K311, KC_NO, K313, K314 }, \
+    { K400, K401,   K402, KC_NO, K404, KC_NO, K406, KC_NO, K408, KC_NO, K410, K411, K412,  K413, K414 }, \
+}
+
+#define LAYOUT_60_ansi( \
+    K000, K001, K002, K003, K004, K005, K006, K007, K008, K009, K010, K011, K012,       K014, \
+    K100,       K102, K103, K104, K105, K106, K107, K108, K109, K110, K111, K112, K113, K114, \
+    K200,       K202, K203, K204, K205, K206, K207, K208, K209, K210, K211, K212,       K214, \
+    K300,       K302, K303, K304, K305, K306, K307, K308, K309, K310, K311,       K313,       \
+    K400, K401, K402,                   K406,                   K410, K411,       K413, K414 \
+) \
+{ \
+    { K000, K001,  K002, K003,  K004,  K005,  K006, K007,  K008, K009,   K010, K011, K012,  KC_NO,  K014 }, \
+    { K100, KC_NO, K102, K103,  K104,  K105,  K106, K107,  K108, K109,   K110, K111, K112,  K113,   K114 }, \
+    { K200, KC_NO, K202, K203,  K204,  K205,  K206, K207,  K208, K209,   K210, K211, K212,  KC_NO,  K214 }, \
+    { K300, KC_NO, K302, K303,  K304,  K305,  K306, K307,  K308, K309,   K310, K311, KC_NO, K313,   KC_NO }, \
+    { K400, K401,  K402, KC_NO, KC_NO, KC_NO, K406, KC_NO, KC_NO, KC_NO, K410, K411, KC_NO, K413,   K414 }, \
+}
+
+#define LAYOUT_60_ansi_split_bs_rshift( \
+    K000, K001, K002, K003, K004, K005, K006, K007, K008, K009, K010, K011, K012, K013, K014, \
+    K100,       K102, K103, K104, K105, K106, K107, K108, K109, K110, K111, K112, K113, K114, \
+    K200,       K202, K203, K204, K205, K206, K207, K208, K209, K210, K211, K212,       K214, \
+    K300,       K302, K303, K304, K305, K306, K307, K308, K309, K310, K311,       K313, K314, \
+    K400, K401, K402,                   K406,                   K410, K411,       K413, K414 \
+) \
+{ \
+    { K000, K001,  K002, K003,  K004,  K005,  K006, K007,  K008, K009,   K010, K011, K012,  K013,   K014 }, \
+    { K100, KC_NO, K102, K103,  K104,  K105,  K106, K107,  K108, K109,   K110, K111, K112,  K113,   K114 }, \
+    { K200, KC_NO, K202, K203,  K204,  K205,  K206, K207,  K208, K209,   K210, K211, K212,  KC_NO,  K214 }, \
+    { K300, KC_NO, K302, K303,  K304,  K305,  K306, K307,  K308, K309,   K310, K311, KC_NO, K313,   K314 }, \
+    { K400, K401,  K402, KC_NO, KC_NO, KC_NO, K406, KC_NO, KC_NO, KC_NO, K410, K411, KC_NO, K413,   K414 }, \
+}
+
+#define LAYOUT_60_iso( \
+    K000, K001, K002, K003, K004, K005, K006, K007, K008, K009, K010, K011, K012,       K014, \
+    K100,       K102, K103, K104, K105, K106, K107, K108, K109, K110, K111, K112, K113, K114, \
```

Comment:
This macro defines 63 keys, but a proper `LAYOUT_60_iso` is 62.

You've added `K213` for the `KC_NUHS` key, but `K114` (ANSI Backslash) is still present here. This is only physically possible if you're running an Enter key that is 1u high and 1.25u wide. I don't believe `K114` belongs here.

```suggestion
    K100,       K102, K103, K104, K105, K106, K107, K108, K109, K110, K111, K112, K113,       \
```